README.md

```
@@ -95,23 +95,27 @@ make setup-postgres
 make setup-redis
 ```
 
-#### PostgreSQL and Redis elsewhere
+#### Configure environment variables
 
-If you choose to run PostgreSQL or Redis anywhere other than those containers, then you may need to customize environment variables for quipucords. That may include, but not be limited to, setting the following environment variables wherever quipucords will run:
+Regardless of how you run PostgreSQL and Redis, you must export environment variables so quipucords can connect to them. The following values match the containers created by `make setup-postgres` and `make setup-redis`:
 
 ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)